cupyx/_ufunc_config.py

```diff
@@ -76,8 +90,12 @@ def seterr(*, divide=None, over=None, under=None, invalid=None, linalg=None):
     if linalg is not None:
         if linalg not in ('ignore', 'raise'):
             raise NotImplementedError()
-
-    old_state = geterr()
+    if fallback_mode is not None:
+        if fallback_mode in ['print', 'warn', 'ignore', 'raise']:
```

Comment:
How about raising `NotImplementedError` for `log`?

Reply:
Done. Same for `call`.